app/client/src/components/shared/Histogram.tsx

```diff
@@ -54,6 +56,8 @@ export default function Histogram({
   yTitle = null,
   yMin = 0,
 }: Props) {
+  const [uuid] = useState<string>(generateUUID());
```

Comment:
To avoid the added library, a ref can be used here instead:
```suggestion
  const chartRef = useRef<HighchartsReact.RefObject>(null);
```